老國音.js

```diff
@@ -0,0 +1,291 @@
+/* 推導老國音
+
+音系：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E9%9F%B3%E7%B4%A0
+音變規則：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E8%88%87%E5%BB%A3%E9%9F%BB%E5%B0%8D%E6%AF%94
+字音來源：https://github.com/baopaau/rime-bepemefeve/blob/main/bepemefeve.dict.yaml
+注意「僞老國音」：https://www.bilibili.com/read/cv17377530/
+
+@author 307587 */
+
+/** @type { 音韻地位['屬於'] } */
+const is = (...x) => 音韻地位.屬於(...x);
+/** @type { 音韻地位['判斷'] } */
+const when = (...x) => 音韻地位.判斷(...x);
+
+if (!音韻地位) return [
+  ['注音符號\n否的話就用國際音標', true],
+  ['ㄬㄭ|ㄬ、ㄭ音標\n啟用「注音符號」時無效', [2, 'ȵ、ɿ、ʅ', 'ɲ、ɹ̩、ɻ̍']],
+  ['介母', [1,
+    { text: '元音：i u y', value: '元音' },
+    { text: '半元音：j w ɥ', value: '半元音' },
+  ], { hidden: 選項.注音符號 === true }],
+  ['ㄚ|ㄚ元音\nㄧㄞ：寬式/iai/、嚴式[iæi]\nㄧ、ㄩ後的ㄢ：寬式/-an/、嚴式[-ɛn]', [2, '嚴式', '寬式']],
+  ['ㄨㄣ|舌冠聲母後的ㄨㄣ讀音\n趙元任《國音新詩韻》：「ㄨㄣ」也可以一律照樣拼念作「ㄨㄜㄋ」，但是在「ㄉ，ㄊ，ㄋ，ㄌ；ㄗ，ㄘ，ㄙ」七個聲母後頭可以省去「ㄜ」音念「ㄨㄋ」', [2, 'un', 'uən']],
```

Comment:
```suggestion
  ['ㄨㄣ|\n趙元任《國音新詩韻》：「ㄨㄣ」也可以一律照樣拼念作「ㄨㄜㄋ」，但是在「ㄉ，ㄊ，ㄋ，ㄌ；ㄗ，ㄘ，ㄙ」七個聲母後頭可以省去「ㄜ」音念「ㄨㄋ」', [2, 'un', 'uən'], { hidden: 選項.注音符號 !== false }],
```

[un] vs [uən] 是对这个韵母记音处理的区别，不是为了表示不同声母后的区别，所以这个选项应当对所有声母都适用